pkg/compactor/bucket_compactor_test.go

```diff
@@ -280,3 +280,71 @@ func TestConvertCompactionResultToForEachJobs(t *testing.T) {
 	require.Equal(t, ulidWithShardIndex{ulid: ulid1, shardIndex: 1}, res[0])
 	require.Equal(t, ulidWithShardIndex{ulid: ulid2, shardIndex: 3}, res[1])
 }
+
+func TestCompactedBlocksTimeRangeVerification(t *testing.T) {
+	tests := map[string]struct {
+		minTime        int64
+		maxTime        int64
```

Comment:
[nit] Just to clarify it.

```suggestion
		compactedMinTime        int64
		compactedMaxTime        int64
```